.github/workflows/ci.yml

```diff
@@ -1,123 +0,0 @@
-name: CI
```

Comment:
@ankushdharkar this should not be deleted. Also you should add to the `test-packages` group of tests.  

```yml
      - name: Run ember-cli-fastboot-testing-app Tests        
      	run: yarn workspace ember-cli-fastboot-testing-app test:ember
```

Reply:
Thanks. Yeah, I messed up during the merge :(